dataactcore/interfaces/function_bag.py

```diff
@@ -73,11 +73,11 @@ def checkPermissionByBitNumber(user, bitNumber):
     # This could likely be simplified, but since we're moving towards using MAX for authentication,
     # it's not worth spending too much time reworking.
 
-    if user.permissions == None:
+    if user.permissions is None:
         # This user has no permissions
         return False
     # First get the value corresponding to the specified bit (i.e. 2^bitNumber)
-    bitValue = 2 ** (bitNumber)
+    bitValue = 2 ** bitNumber
```

Comment:
Also, thanks for taking the extra time to remove the extra `()` throughout...a big help for overall readability.